runtime/parachains/src/paras_inherent.rs

```diff
@@ -274,7 +472,7 @@ pub mod pallet {
 			} = <inclusion::Pallet<T>>::process_candidates(
 				parent_storage_root,
 				backed_candidates,
-				<scheduler::Pallet<T>>::scheduled(),
+				scheduled,
 				<scheduler::Pallet<T>>::group_validators,
 			)?;
```

Comment:
This is the second and last occasion the block can error, but given our latest impl it will not. Have to double check.

Reply:
`process_candidates` can still error. I briefly mentioned this yesterday, but I am wondering if we should also modify process candidates so it will just skip candidates and not error if there are any issue

Reply:
I think modifying it might be out of scope of this PR, but from my understanding of things it could make the liveliness of the relay chain more resilient if we simply skip candidates that have any error instead of error'ing out of the whole `enter` inherent